ompi/mpi/man/man3/MPI_Win_create_dynamic.3in

```diff
@@ -80,7 +83,14 @@ If set to \fIsame_op\fP, the implementation will assume that all concurrent
 accumulate calls to the same target address will use the same
 operation.  If set to \fIsame_op_no_op\fP, then the implementation will
 assume that all concurrent accumulate calls to the same target address
-will use the same operation or MPI_NO_OP.  The default is \fIsame_op_no_op\fP.
+will use the same operation or MPI_NO_OP. The default is \fIsame_op_no_op\fP.
+.sp
+
+.SH NOTES
+Since dynamically attaching memory to windows are not collective calls, one have to communicate the actual address at the target using MPI_Get_address and some communication.
```

Comment:
```suggestion
Since dynamically attaching memory to windows is not a collective call, one has to communicate the actual address at the target using \fIMPI_Get_address\fP and some communication.
```